frontend/XCITE/Nodes/RegistrationLevel.qml

```diff
@@ -43,195 +40,98 @@ Controls.Diode {
+        width: 312
+        height: 397
         radius: 5
         color: "#2A2C31"
-        ColumnLayout {
-            anchors.top: parent.top
-            anchors.left: parent.left
-            anchors.right: parent.right
-            anchors.topMargin: 65
-            anchors.leftMargin: 15
-            anchors.rightMargin: 15
-            Label {
-                id: label1
-                font.weight: Font.Light
-                font.pixelSize: 16
-                bottomPadding: 5
-                color: "#d5d5d5"
-                text: {
-                    if (level1 == 3)
-                        qsTr("      Earnings:                    Low")
-                    if (level1 == 2)
-                        qsTr("      Earnings:               Medium")
-                    if (level1 == 1 || level1 == 0)
-                        qsTr("      Earnings:                    High")
-                }
-                Image {
-
-                    id: diamond
-                    visible: true
-                    anchors {
-                        left: parent.left
-                    }
-                    source: "../../icons/icon-diamond.svg"
-                }
-            }
-            Label {
-                id: label2
-                font.weight: Font.Light
-                font.pixelSize: 16
-                bottomPadding: 5
-                color: "#d5d5d5"
-                text: {
-                    if (level1 == 3)
-                        qsTr("      Network Stake:            Low")
-                    if (level1 == 2)
-                        qsTr("      Network Stake:       Medium")
-                    if (level1 == 1 || level1 == 0)
-                        qsTr("      Network Stake:            High")
-                }
-                Image {
-
-                    id: network
-                    visible: true
-                    anchors {
-                        left: parent.left
-                    }
-                    source: "../../icons/icon-network.svg"
-                }
-            }
-            Label {
-                id: label3
-                font.weight: Font.Light
-                font.pixelSize: 16
-                bottomPadding: 15
-                color: "#d5d5d5"
-                text: {
-                    if (level1 == 3)
-                        qsTr("      Transfer Rate:              Low")
-                    if (level1 == 2)
-                        qsTr("      Transfer Rate:         Medium")
-                    if (level1 == 1 || level1 == 0)
-                        qsTr("      Transfer Rate:              High")
-                }
-                Image {
-
-                    id: transfer
-                    visible: true
-                    anchors {
-                        left: parent.left
-                    }
-                    source: "../../icons/icon-transfer.svg"
-                }
-            }
+        earningsText: qsTr("      Earnings:")
```

Comment:
Ideally try not to pad strings with spaces, this'll only bite you later when things are resized. Text inside `qsTr()` will ultimately need to be matched by the i18n/language support so should be strings that make sense to translate.